test/lint/lint-python.sh

```diff
@@ -44,7 +44,6 @@ enabled=(
     E743 # do not define functions named "l", "O", or "I"
     E901 # SyntaxError: invalid syntax
     E902 # TokenError: EOF in multi-line string
-    F401 # module imported but unused
```

Comment:
why remove this check?

Reply:
https://github.com/bitcoin/bitcoin/pull/18210#issuecomment-639266292

Reply:
But why does travis not print this warning message?

Reply:
False positive warnings due to the outdated local `flake8` version:
```
$ pip3 show flake8 | grep Version
Version: 3.5.0
```

Going to fix this pull now.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19172#issuecomment-639475496).